UniMath/CategoryTheory/.package/files

```diff
@@ -18,13 +18,13 @@ limits/initial.v
 limits/graphs/initial.v
 limits/zero.v
 limits/graphs/zero.v
+limits/bincoproducts.v
 limits/coproducts.v
-limits/arbitrary_coproducts.v
-limits/graphs/coproducts.v
+limits/graphs/bincoproducts.v
```

Comment:
What is camelcase? How does one know how to write in it when one need to import bincoproducts.v?

> On Jun 19, 2016, at 3:51 PM, Benedikt Ahrens <notifications@github.com> wrote:
> 
> In UniMath/CategoryTheory/.package/files <https://github.com/UniMath/UniMath/pull/367#discussion_r67622785>:
> 
> >  limits/coproducts.v
> > -limits/arbitrary_coproducts.v
> > -limits/graphs/coproducts.v
> > +limits/graphs/bincoproducts.v
> Would a camelcase BinCoproducts.v be easier to read?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/UniMath/UniMath/pull/367/files/c24a07114befa20c3c9fa4bda4c74e9d4e816b56#r67622785>, or mute the thread <https://github.com/notifications/unsubscribe/AAZ4Swpvrz3agdAGNAqIu0NfmXuv6i6Oks5qNZ22gaJpZM4I5Odh>.
>